src/main/java/seedu/address/model/module/exceptions/DuplicateModuleException.java

```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.module.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Modules (Modules are considered duplicates if they have the same
+ * code).
+ */
+public class DuplicateModuleException extends RuntimeException {
+    public DuplicateModuleException() {
+        super("Operation would result in duplicate persons");
```

Comment:
I think it is better if the error message is "Operation would result in duplicate modules".

Reply:
oya lol.